Detect devices for brands: Samsung, Vivo, iHunt, Huawei, Ritmix, Ross&Moor

sanchezzzhak opened the pull request.

sanchezzzhak committed:
feat: added detect Lenovo E10 (TB-X104X, TB-X104L, TB-X104F)

sanchezzzhak committed:
feat: added detect iHunt S10 Tank 2019

sanchezzzhak committed:
feat: added detect Huawei MediaPad M6 8.4 (VRD-W09)

sanchezzzhak committed:
feat(devices): added detect tests devices
Ritmix: RMD-1026, RMD-1028, RMD-1058,  RMD-1059, RMD-726
Ross&Moor: RMD-974R, RMD-600
issue #6041

sanchezzzhak committed:
upgrade fork;

sanchezzzhak committed:
feat: added detect Samsung GALAXY A10s (SM-A107F, SM-A107M)

sanchezzzhak committed:
feat: added detect Samsung GALAXY M30s (SM-M307F, SM-M307FN)

sanchezzzhak committed:
feat(devices): added detect devices
Samsung GALAXY J2 Pure (SM-J260AZ)
Samsung GALAXY J2 Core (SM-J260F, SM-J260G, SM-J260M, SM-J260T1, SM-J260A)

sanchezzzhak committed:
feat(devices): added detect devices
Vivo Y53i (1606)
Vivo V9 (1723)
Vivo V17 Pro (1909)
Vivo Y83 Pro (1726)

sanchezzzhak committed:
feat(devices): added detect devices
Amazon Echo (AEOBC, AEOKN)
issue #5939

sanchezzzhak committed:
feat(devices): added detect devices
Vivo Y55l (1603)
Vivo Y55s (1610)
Vivo V7 (1718)
Vivo Y65 (1719)
Vivo V9 Youth (1727)
Vivo Y71i (1801)
Vivo V11 Pro (1804)
Vivo Nex LTE (1805)
Vivo V11i (1806)
Vivo Y81 (1808)
Vivo Y91 (1811)
Vivo V15 Pro (1818)
Vivo Y91i (1820)
Vivo Y17 (1902)

sanchezzzhak committed:
feat(devices): added detect devices
Vivo V5 Plus (1611)
Vivo X9 Plus
Vivo 20 Plus
Vivo V5 Lite (1609)
Vivo V5s (1713, 1612)
Vivo V7 Plus (1716)
Vivo V5 (1601)
Vivo Y69 (1714)
Vivo X20 (1721)
Vivo Y71 (1724)
Vivo X21 (1725)
Vivo Y81 (1803, 1808i)
Vivo Y95 (1807)
Vivo Y93 (1814, 1815)

sgiehl commented:
@sanchezzzhak PR looks fine. Could you please resolve the conflicts? Would merge this afterwards

sanchezzzhak committed:
Merge remote-tracking branch 'upstream/master' into pr_device_26
Resolve merge conflicts in Tests/fixtures/smartphone-12.yml

sanchezzzhak committed:
feat(devices): added detect devices
Motorola Moto G5s Plus (XT1804, XT1803, XT1805)

issue #6041

sanchezzzhak committed:
feat(devices): added detect devices
Motorola Moto G5 (XT1670)
Motorola Moto G5 Plus (XT1685, XT1687)
MicroMax YU Yureka (YU5530, YU5040)

issue #6041

sanchezzzhak committed:
feat(devices): added detect device Mobicel R1

issue #6067 #6041

sanchezzzhak committed:
feat(devices): added detect device Mobiistar Zumbo S2
issue #6041

sanchezzzhak committed:
feat(devices): added detect devices
Insignia models: NS-P11A8100, NS-P10A8100, NS-P10A6100, NS-P16AT08, NS-P16AT10, NS-P16AT785HD, NS-P08A7100, NS-P10A7100;
NOA models: Next SE;

issue #6041

sgiehl committed:
Merge branch 'master' into pr_device_26
